types/onecolor/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for onecolor 3.1
+// Project: onecolor
```

Comment:
As there are multiple projects, can you refere via link which one these types refer to?
sample:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/base64-js/index.d.ts#L2

Reply:
Thanks @peterblazejewicz, ref link added in this [commit](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41253/commits/271db78639557c73559e4686ec4fcf29fb892ea8)